app/src/backend/UserService.ts

```diff
@@ -159,16 +159,70 @@ export default class UserService {
         type: QueryTypes.SELECT,
       },
     )) as { inventory_id: string }[];
-    if (!inventory) {
-      throw new createHttpError.NotFound("Inventory not found");
+
+    if (inventory) {
+      await db.models.User.update(
+        {
+          defaultInventoryId: inventory?.inventory_id,
+        },
+        { where: { userId } },
+      );
+      return inventory?.inventory_id;
     }
-    await db.models.User.update(
-      {
-        defaultInventoryId: inventory?.inventory_id,
+
+    // throw new createHttpError.NotFound("Inventory not found");
+
+    const adminData = await db.models.OrganizationAdmin.findOne({
+      where: {
+        userId: userId,
       },
-      { where: { userId } },
-    );
-    return inventory?.inventory_id;
+      include: {
+        model: db.models.Organization,
+        as: "organization",
+        include: {
+          model: db.models.Project,
+          as: "projects",
+          include: [
+            {
+              model: db.models.City,
+              as: "cities",
+              include: [
+                {
+                  model: db.models.Inventory,
+                  as: "inventories",
+                  attributes: ["inventoryId"],
+                },
+              ],
+            },
+          ],
+        },
+      },
```

Comment:
To resolve the build error, pass an array instead of an object here.
```suggestion
      include: [{
        model: db.models.Organization,
        as: "organization",
        include: {
          model: db.models.Project,
          as: "projects",
          include: [
            {
              model: db.models.City,
              as: "cities",
              include: [
                {
                  model: db.models.Inventory,
                  as: "inventories",
                  attributes: ["inventoryId"],
                },
              ],
            },
          ],
        },
      }],
```